azure-pipelines.vrt-pr.yml

```diff
@@ -28,6 +28,8 @@ jobs:
 
```

Comment:
## [🕵 FluentUI-v0 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27057&commitId=62121b92c934668a259b4e2cdbf75c9ed02331dc&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1258 screenshots unchanged, 0 screenshots with different dimensions and 2 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|List.List Example Selectable Teams.Focuses item.chromium.png|117| Changed|
|List.List Example Selectable Teams.chromium.png|117| Changed|

</details>


<div id="vrtCommentFluentUI-v0"/>